pkgs/development/python-modules/verboselogs/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "verboselogs";
+  version = "1.7";
+
+  src = fetchFromGitHub {
+    owner  = "xolox";
+    repo   = "python-verboselogs";
+    rev    = version;
+    sha256 = "10jzm8pkl49as4y2zyiidmfqqj5zmqg3p73jvx4lfxi0gmp1vhl5";
+  };
+
+  # do not run checks, requires old version of astroid
```

Comment:
if it's the same minor version, you can try just editing the version bounds. Often times people are over constraining about version bounds.

However, they require < astroid 1.3.2.....
either you can add a pinned version under the python-modules/verboselogs/ or....... since it's just a test dependency... skip the test.

In this case I'll lean on the side of just saying, "it's as safe as pip" without tests.

Reply:
They have changed how the package works, so the test will not run at all.

Reply:
yea, i would just skip the tests then, too much effort to pick up from the author.